src/data/reducers/courseInfo.js

```diff
@@ -31,6 +35,27 @@ function courseInfo(state = initialState, action) {
         isFetching: true,
         error: null,
       });
+    case CREATE_COURSE:
+      return Object.assign({}, state, {
+        data: {},
+        courseCreated: false,
+        isFetching: false,
+        error: null,
+      });
```

Comment:
1) courseCreated... It's used as a signal to switch pages, yeah? I guess that's the right pattern? Feels weird to me, but alright.

2) You're setting isFetching but don't care about isFetching. You can just leave it out, and it won't be touched. (the current state will come in via the `state` argument to `Object.assign` and stay the same if you don't override it)

Reply:
For courseCreated, I would be interested to hear about another approach you might have in mind. I was using it to signal the course was created successfully.

Reply:
I was able to update this and make it simpler I think.  Take a look at the PR now.

Reply:
Oh nice, moving the push to the fetch callback does simplify the component caring about that state. It is now maybe a bit presumptiuous of the fetch callback to assume the page wants to be switched? But we only call it from the one place for now anyway.

I still think it's unnecessary to set isFetching, since that's not part of the create-course action flow. But it isn't harmful.